python3/raygun4py/raygunmsgs.py

```diff
@@ -185,6 +188,16 @@ def __init__(self, exc_type, exc_value, exc_traceback, options):
     def get_classname(self):
         return self.className
 
+    def _get_frames(self, exc_traceback):
+        if self._is_stack_frame_type(exc_traceback):
+            return exc_traceback
+        else:
+            return inspect.getinnerframes(exc_traceback)
+
+    def _is_stack_frame_type(self, exc_traceback):
+        matches = re.findall(self.INSPECT_FRAME_TYPE_SIGNATURE, str(exc_traceback).lower())
```

Comment:
Is the `findall` method necessary here? Could we use a cheaper method like `match()` instead?